### PR TITLE
Bigtable Row Data snippets

### PR DESCRIPTION
**Pull request by sangramql**

### CODE REVIEW

**Comment by sangramql** [comment]
@tseaver @sduskis @AVaksman
Please review the updated PR.

**Comment by sangramql** [comment]
@tseaver @sduskis This is updated PR and ready to merge i suppose.

File: bigtable/docs/snippets_table.py [hunk 1]
```diff
@@ -403,5 +412,84 @@ def test_bigtable_table_row():
     table.truncate(timeout=300)
 
 
-if __name__ == "__main__":
+def test_bigtable_row_data_cells_cell_value_cell_values():
+
+    value = b'value_in_col1'
+    row = Config.TABLE.row('row_key_1')
```
Comment (billyjacobson) [inline bigtable/docs/snippets_table.py]: should this rowkey be a bytes value as well?

**Comment by crwilcox** [comment]
Ported to #7019

**Pull request closed without merging**